timetracker/entries/forms.py

```diff
@@ -1,24 +1,26 @@
 from django import forms
 from django.utils import timezone
 
-from .models import Project, Client
+from .models import Project, Client, Entry
 
 
-class ClientForm(forms.Form):
-    name = forms.CharField()
+class ClientForm(forms.ModelForm):
+    class Meta:
+        model = Client
+        fields = ('name',)
 
 
-class ProjectForm(forms.Form):
-	name = forms.CharField()
-	client = forms.ModelChoiceField(queryset=Client.objects.all())
+class ProjectForm(forms.ModelForm):
+    class Meta:
+        model = Project
+        fields = ('name', 'client')
 
 
-class EntryForm(forms.Form):
-    start = forms.DateTimeField(label="Start Time", help_text="Format: 2006-10-25 14:30")
-    stop = forms.DateTimeField(label="End Time", help_text="Format: 2006-10-25 14:30")
-    project = forms.ModelChoiceField(queryset=Project.objects.all())
-    description = forms.CharField()
-
+class EntryForm(forms.ModelForm):
+    class Meta:
+        model = Entry
+        fields = ('start', 'stop', 'project', 'description')
```

Comment:
You're loosing the help text for the form fields here. Either define them as `help_texts` on `EntryForm.Meta` or as part of the respective model field.